src/lib/components/WellCompletions/utils/sort.ts

```diff
@@ -12,9 +12,9 @@ const createSortKeyFunction = (
             return (well) => well.name;
         case SORT_BY_STRATIGRAPHY_DEPTH:
             return (well) =>
-                well.zoneIndices.find(
-                    (_, index) => well.completions[index] > 0
-                );
+                well.completions.find(
+                    (_, index) => well.completions[index]!.open > 0
```

Comment:
This triggers an `Forbidden non-null assertion` warning.

Reply:
Yes I saw that too. Not sure what's the best way to fix the warning as well.completions[index] is obviously not undefined.

Reply:
Added a comment to disable the warning for that line